packages/scroll/README.md

```diff
@@ -124,7 +124,6 @@ Get an `{ x: number, y: number }` object of element/window scroll position.
 
 _PRs Welcome :)_
 
-- `createScrollTo` - A primitive to support scroll to a target
```

Comment:
scroll*ing* to a target

Reply:
Thanks for the review, does that mean replacing "scroll" with "scrolling"?